builders/integrations/indexers/subquery.md

```diff
@@ -50,216 +53,332 @@ In general, you will need to:
 
 3. You'll be prompted to answer a series of questions:
 
-    1. For the **Select a network family** question, you can choose **Substrate**
+    1. For the **Select a network family** question, although Moonbeam is EVM compatible, the Moonbeam templates are under the **Polkadot** family, so you can choose **Polkadot**
 
-        ![Select Moonbeam](/images/builders/integrations/indexers/subquery/subquery-1.png)
+        ![The SubQuery CLI choose a network family menu.](/images/builders/integrations/indexers/subquery/new/subquery-1.png)
 
-    2. The next screen will prompt you to **Select a network**. At time of writing this guide, Moonriver was the only option. You can go ahead and choose **Moonriver**, and it can be adapted for Moonbeam or Moonbase Alpha
+    2. The next screen will prompt you to **Select a network**. At the time of writing this guide, Moonbeam and Moonriver were the only options
 
-        ![Select moonbeam-starter](/images/builders/integrations/indexers/subquery/subquery-2.png)
+        !!! note
+            To build a project on Moonbase Alpha, you can select either network and adapt it later on
 
-    3. You'll be prompted to **Select a template project**. You can choose between the EVM starter project or creating a project from a git endpoint. Since this guide will be based off of the Moonriver EVM starter project, you can select **moonriver-evm-starter**
+        ![The SubQuery CLI choose a network menu.](/images/builders/integrations/indexers/subquery/new/subquery-2.png)
 
-        ![Select moonbeam-starter](/images/builders/integrations/indexers/subquery/subquery-3.png)
+    3. You'll be prompted to **Select a template project**. Depending on the network you chose in the prior step, the template options may vary
 
-    4. The starter project will be cloned, and then you will be prompted to answer a few more questions. For these, you can just hit enter and accept the default or customize them as you see fit
+        === "Moonbeam"
 
-        ![Create project](/images/builders/integrations/indexers/subquery/subquery-4.png)
+            |             Template             |                                                                             Description                                                                              |
```

Comment:
![Screenshot 2023-12-19 at 7 43 48 PM](https://github.com/moonbeam-foundation/moonbeam-docs/assets/5834262/05503445-47bd-4a20-85e1-7775bfd379ea)
If the heading and subheading could be aligned that would be ideal, if not no worries

Reply:
Yeah, unfortunately, doing this across the board is not easy because there can be N number of tabs and N number of table headers